src/assets/i18n/de.json

```diff
@@ -150,17 +150,17 @@
 		"editMemberInstitute": "Bearbeite die Mitglieder der Institution.",
 		"membersHaveAccess": "Diese Mitglieder haben Zugriff auf die Institution.",
 		"addMember": "Mitglied hinzufügen",
-		"addMember_text": "Neue Mitarbeiter müssen ein OEB-Konto anlegen und verifizieren, bevor sie zu einem Emittenten hinzugefügt werden.",
+		"addMember_text": "Neue Mitarbeiter müssen ein OEB-Konto anlegen und verifizieren, bevor sie zu einem Issuer hinzugefügt werden.",
 		"addMember_chooseRole": "Rolle wählen",
-		"addMember_owner": "Owner",
+		"addMember_owner": "Eigentümer",
 		"addMember_ownerText": "Möglichkeit, Mitarbeiter hinzuzufügen und zu entfernen. Volle Rechte zum Erstellen, Löschen und Verleihen von Abzeichen. Möglichkeit, Ausstellerdetails zu bearbeiten.",
-		"addMember_editor": "Editor",
+		"addMember_editor": "Herausgeber",
```

Comment:
I think we dont need this translation for ```Editor```.  The word ```Editor``` also exists in german and is more fitting in this context than ``` Herausgeber``` imo

Reply:
I've updated it.